pkg/profiler/profiler.go

```diff
@@ -526,6 +536,11 @@ func (p *CgroupProfiler) profileLoop(ctx context.Context, captureTime time.Time)
 		level.Warn(p.logger).Log("msg", "failed to clean BPF maps", "err", err)
 	}
 
+	ksymCacheStats := p.ksymCache.Stats
+	level.Debug(p.logger).Log("msg", "Kernel symbol cache stats", "stats", ksymCacheStats.String())
+	p.metrics.ksymCacheHitRate.WithLabelValues("hits").Add(float64(ksymCacheStats.Hits))
+	p.metrics.ksymCacheHitRate.WithLabelValues("total").Add(float64(ksymCacheStats.Total))
```

Comment:
This is actually not correct regarding Prometheus conventions. It's like double counting. I would expect to see "hit", "miss" label values and if I would like to have the total I would just skip specifying the label value for the metric.

Reply:
I see, I will update the PR to submit the hits with the 'hits' label, and the total without labels. 

Never used Prometheus so I am not sure, but in the UI is it going to be clear that the 'total' is the one without a label?

Reply:
You don't need to increase total without label. Labels are for slicing and dicing the a whole set, so a query without specifying a label will give you the total set out of the box. For example, when you have "hits" + "miss" = "total". With your proposed way, it would be  2 x "hits" = "total", so it's false.

Check this presentation, it would help https://promcon.io/2017-munich/slides/best-practices-and-beastly-pitfalls.pdf